LS_Miroshnichenko/Extensions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Extensions.swift
+//  LS_Miroshnichenko
+//
+//  Created by Мирошниченко Марина on 03.03.2020.
+//  Copyright © 2020 Мирошниченко Марина. All rights reserved.
+//
+
+import Foundation
+
+// include hidden files
+extension FileManager {
+    func urls(for directory: String, skipsHiddenFiles: Bool = true ) -> [URL]? {
+        return try? FileManager.default.contentsOfDirectory(at: URL(fileURLWithPath: directory), includingPropertiesForKeys: nil, options: skipsHiddenFiles ? .skipsHiddenFiles : [] )
+    }
+}
+
+extension URL {
+var attributes: [FileAttributeKey : Any]? {
+        do {
+            return try FileManager.default.attributesOfItem(atPath: path)
+        } catch let error as NSError {
+            print("FileAttribute error: \(error)")
+        }
+        return nil
+    }
```

Comment:
I think it's better to make this extension in `FileManager` also. The thing is URL can hold not only local file URL, but also web URL, while working with FileManager we can be sure we're working with files. Also it's not very good to have such strong coupled the dependency (`FileManager.default`)
Alternatively this var could be renames to something like `fileAttributes` to make it more clear what do we want of it